core/controllers/collection_editor.py

```diff
@@ -172,12 +195,19 @@ def put(self, collection_id):
             raise self.InvalidInputException(e)
 
         # Retrieve the updated collection.
+        collection = (
+            collection_services.get_collection_by_id(collection_id))
+
         collection_dict = (
             summary_services.get_learner_collection_dict_by_id(
                 collection_id, self.user_id, allow_invalid_explorations=True))
 
-        # Send the updated collection back to the frontend.
-        self.values.update(collection_dict)
+        # return updated collection
```

Comment:
When adding comments, it's generally a good idea to keep them formatted in a consistent way with neighboring comments (e.g. capitalizing them and ending with punctuation, in this case).